cursorless-talon/src/public_api/public_api_v1.py

```diff
@@ -0,0 +1,75 @@
+from typing import Any, Optional, Union
+
+from talon import Module, actions
+
+from ..targets.target_types import (
+    CursorlessTarget,
+    ListTarget,
+    PrimitiveTarget,
+    RangeTarget,
+    RangeTargetType,
+)
+from .extract_decorated_marks import extract_decorated_marks
+
+mod = Module()
+
+
+@mod.action_class
+class MiscActions:
+    def cursorless_v1_extract_decorated_marks(capture: Any) -> list[dict]:
```

Comment:
This is currently used as follows: https://github.com/pokey/wax_talon/blob/4bd7d9b9112a4172d4fc46fe5644e33b8b1340b7/recorders/cursorless_recorder.py#L151-L159.  The `parsed` argument is expected to be `list(j["parsed"])`, where `j` is the argument to an `on_phrase` callback.  I wonder if we want to expose something that operates directly on the phrase object rather than requiring the pre-processing 🤔.  This function does work on the output of any of our spoken form captures, so I think may still be a reasonable api